lib/protoc.dart

```diff
@@ -15,15 +15,15 @@ import 'src/dart_options.pb.dart';
 import 'const_generator.dart' show writeJsonConst;
```

Comment:
Moving and/or reformatting files means people with an uncommitted patch in progress will have to redo their work because merges will fail. For example, see: cl/123879507

It doesn't seem urgent, so I'd rather wait until there's no work in progress. (It's often the case but this time your timing is unfortunate.)